src/views/KeyboardManager.tsx

```diff
@@ -14,6 +14,13 @@ export default class KeyboardManager extends React.Component<Props> {
   // When a gesture didn't change the tab, we can restore the focused input with this
   private previouslyFocusedTextInput: number | null = null;
   private startTimestamp: number = 0;
+  private keyboardTimeout: NodeJS.Timer | null = null;
+
+  componentWillUnmount = () => {
+    if (this.keyboardTimeout !== null) {
+      global.clearTimeout(this.keyboardTimeout);
```

Comment:
```suggestion
      clearTimeout(this.keyboardTimeout);
```

Reply:
Probably should also clear the time out on beginning of `handlePageChangeStart`, `handlePageChangeConfirm` and `handlePageChangeCancel` in case there is another quick swipe after cancel?